Dockerfile

```diff
@@ -16,3 +16,7 @@ RUN chmod +x /entrypoint.sh
 
 COPY goreleaser_*.apk /tmp/
 RUN apk add --allow-untrusted /tmp/goreleaser_*.apk
+
+RUN mkdir -p $HOME/.docker/cli-plugins/ && \
```

Comment:
I think we can just add `docker-cli-buildx` to the list of packages to install in line 3...

Reply:
even better, I will give it a try and update the PR if it works. Thanks for the hint!

Reply:
works great, I updated the PR

Reply:
You can also use our [buildx-bin](https://github.com/docker/buildx#from-dockerfile) image to add buildx to your image: https://github.com/docker/buildx#from-dockerfile